Formula/arangodb.rb

```diff
@@ -20,7 +20,7 @@ class Arangodb < Formula
 
   depends_on "ccache" => :build
   depends_on "cmake" => :build
-  depends_on "go@1.16" => :build
+  depends_on "go@1.17" => :build
```

Comment:
Will it work with go 1.18?

```suggestion
  depends_on "go" => :build
```

Reply:
This version of ArangoDB Starter was thoroughly tested with Go 1.17 only. So we would like to rely on the specific version here.